### PR TITLE
looking for system architecture for urlSuffix

### PR DESCRIPTION
**Pull request by harsh-not-haarsh**
Checking system architecture using `dpkg` to get urlSuffix for latest release for istio

### CODE REVIEW

**Comment by leecalcote** [comment]
@harsh-not-haarsh I'm sorry that it's taken us so long to review your PR and provide feedback. Thank you for submitting this. 👍 

@kumarabd and @tingkai-zhang are _just_ starting to look at building out a new adapter for Kuma (https://github.com/layer5io/meshery-kuma). I encourage you to join in, @harsh-not-haarsh. It would be fantastic to have you collaborating here.

File: istio/install-istio.go [hunk 1]
```diff
@@ -46,6 +49,18 @@ var (
 	bookInfoInjectHTTPAbortToRatingsForJasonFile = path.Join(basePath, "samples/bookinfo/networking/virtual-service-ratings-test-abort.yaml")
 )
 
+func getArch() string {
+	cmd := exec.Command("dpkg", "--print-architecture")
```
Comment (kumarabd) [inline istio/install-istio.go:53]: Wouldnt this fail on a non-debian container? Just a thought. @harsh-not-haarsh @leecalcote

Comment (kushthedude) [reply istio/install-istio.go:53]: @kumarabd How about having a conditional check for debian based os in the function?

Comment (leecalcote) [reply istio/install-istio.go:53]: We do need to first check for presence of `dpkg` and have a backup plan if unavailable.

**Comment by leecalcote** [comment]
Stale PR

**Pull request closed without merging**